src/chainstate/burn/operations/mod.rs

```diff
@@ -339,6 +357,15 @@ pub enum BlockstackOperationType {
     PegIn(PegInOp),
 }
 
+/// This enum wraps Vecs of a single kind of `BlockstackOperationType`.
+/// This allows `handle_get_burn_ops` to use an enum for the different operation
+///  types without having to buffer and re-structure a `Vec<BlockstackOperationType>`
+///  from a, e.g., `Vec<PegInOp>`
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub enum BurnchainOpsVec {
+    PegIn(Vec<PegInOp>),
+}
```

Comment:
I have two suggestions here:

1. Move this to `src/net/mod`, since it's essentially a helper type for `net::HttpResponseType` and is only used in `src/net`.
2. Nit: Omit "Vec" from the name and just call this `BurnchainOps`.

Reply:
Done in 40c1208c8